perllib/FixMyStreet/Cobrand/Northumberland.pm

```diff
@@ -99,14 +100,19 @@ sub dashboard_export_problems_add_columns {
         my $by = $report->get_extra_metadata('contributed_by');
         my $staff_user = '';
         my $staff_role = '';
+        my $assigned_to = '';
         if ($by) {
             $staff_user = $self->csv_staff_user_lookup($by, $user_lookup);
             $staff_role = join(',', @{$userroles->{$by} || []});
         }
+        if ($report->shortlisted_user) {
```

Comment:
Any change to the CSV export needs to consider the effect on performance. In this case, you are asking for two new DB queries for every row being looked at (first the user planned reports table, then the user table, I can't see any prefetches). As with the `csv_staff_user_lookup` being used just above here, I think if this is a wanted column in the export, you will have to prefetch all the current shortlisted users into a hash, and then here you can then use that stored lookup.
(The other option would be to try and change the main DB query to include the shortlisted user in its results, but that seems more prune to issues.)